lstchain/scripts/lstchain_data_muon_analysis.py

```diff
@@ -74,7 +72,7 @@ def main():
     print("max events: {}".format(args.max_events))
 
     # Camera geometry
-    geom = CameraGeometry.from_name("LSTCam-002")
+    geom = CameraGeometry.from_name("LSTCam-003")
```

Comment:
new camera geometry?

Reply:
this should be the correct one see 
[https://github.com/cta-observatory/ctapipe-extra/pull/25](https://github.com/cta-observatory/ctapipe-extra/pull/25)
I am going to change it also in ctapipe_io_lst